StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -298,6 +304,7 @@ extension VerticalSavedPaymentMethodsViewController: UpdateCardViewControllerDel
         // Create the new button
         let newButton = PaymentMethodRowButton(paymentMethod: updatedPaymentMethod, appearance: configuration.appearance)
         newButton.delegate = self
+        newButton.state = oldButton.previousSelectedState // save the previous state from the old button too
         newButton.state = oldButton.state
```

Comment:
Typo?

Reply:
Technically no, but it does read funny. The second state update will preserve the first state as it's previous state. I've made it easier to read.